src/modules/45-projects-orgs/components/SettingsPageComponent/OrgSettingsPage.tsx

```diff
@@ -136,6 +147,13 @@ export const OrgSettingsPage: React.FC = () => {
               route={routesV2.toSettingsServiceOverrides({ accountId, orgIdentifier, module })}
               hidden={!(isServiceOverridesEnabled && haveCD)}
             />
+            <SettingsResourceCard
+              label={<String stringID="common.webhooks" />}
```

Comment:
Any reason to use `String` here instead of `getString`? Not a blocker, just curious
```suggestion
              label={getString('common.webhooks')}
```